R/snapshot.R

```diff
@@ -45,35 +45,50 @@
 #' @param x Code to evaluate.
 #' @param cran Should these expectations be verified on CRAN? By default,
 #'   they are not, because snapshot tests tend to be fragile because they
-#'   often rely on minor details of dependencies. Even on CRAN, the
-#'   expectations will still fail if `x` errors.
+#'   often rely on minor details of dependencies.
+#' @param error Do you expect the code to throw an error? The expectation
```

Comment:
Can we allow `error` to be a class? Otherwise we'll still need a duplicate `expect_error()` test.